saleor/order/models.py

```diff
@@ -375,8 +375,7 @@ def quantity_unfulfilled(self):
 
     @property
     def is_digital(self) -> bool:
-        """Return true if product variant is a digital type and has assigned
-        digital content"""
+        """Whether the variant is digital and does contain digital content."""
```

Comment:
Not exactly sure about this but I I think this should be correct:
```suggestion
        """Check if a variant is digital and contains digital content."""
```